### PR TITLE
Improvements for volume tests

### PR DESCRIPTION
**Pull request by hmahmood**

### CODE REVIEW

**Comment by matthewavery** [comment]
lgtm

**Comment by jzt** [comment]
LGTM

File: tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot [hunk 1]
```diff
@@ -9,41 +9,49 @@ Simple docker volume create
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create
     Should Be Equal As Integers  ${rc}  0
     Set Suite Variable  ${ContainerName}  unnamedSpecVolContainer
-    Run  docker ${params} run --name ${ContainerName} -d -v ${output}:/mydata busybox /bin/df -Ph
+    ${rc}=  Run And Return Rc  docker ${params} run --name ${ContainerName} -d -v ${output}:/mydata busybox /bin/df -Ph
```
Comment (hmahmood) [inline tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot]: Yes that would be a good idea. Incoming.

**Comment by hmahmood** [comment]
:recycle:

**Pull request merged**